flexo/src/mirror_flexo.rs

```diff
@@ -590,7 +590,27 @@ fn cfs_path_from_pkg_path(path: &Path) -> Option<PathBuf> {
 pub struct DownloadOrder {
     /// This path is relative to the given root directory.
     pub requested_path: StrPath,
-    pub id: Uuid,
+    pub non_cacheable_unique_id: Option<Uuid>,
+}
+
+impl DownloadOrder {
+    pub fn new(requested_path: StrPath) -> Self {
+        if !(requested_path.to_str().ends_with(".db") ||
```

Comment:
I'd prefer it if we still have this entire `if` condition inside a function named `is_cacheable`, just to make our intent a more obvious: Files are considered cacheable if and only if their name ends with one of those strings.